test/running-path/invalid/not_unique_CS.yaml

```diff
@@ -0,0 +1,14 @@
+%YAML 1.2
+---
+schema: https://railtoolkit.org/schema/running-path.json
+schema_version: "2024.07"
+paths:
+  - name: test
+    id: 1
```

Comment:
This errors because id is not a string, but a number.
In this test duplicate CS should be tested